schemas/v2/index.json

```diff
@@ -516,6 +516,108 @@
             ],
             "additionalProperties": false
         },
+        "BitbucketConfig": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "const": "bitbucket",
+                    "description": "Bitbucket configuration"
+                },
+                "countMisc": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Count watchers and forks"
```

Comment:
Still reading, but curious - what does this setting do?

Reply:
I added these counts for parity with the other services, although bitbucket doesn't seem to do stars the way github does.

Against bitbucket cloud I was seeing rate limiting of API calls when watchers and forks were counted for a large number of repos.  I made it configurable for the time being, since I don't need the info.

In the server API, I don't see these counts, so they will be zero in that case.